oaknational/Oak-Web-Application
# Feat/lesq 430/promote search api [LESQ-430]

## Description

Music year: 1961

- Removes the feature flag and uses 2023 api on all searches


## How to test

1. Go to https://deploy-preview-2153--oak-web-application.netlify.thenational.academy
2. Search for 'macbeth' for example
3. You should see 'new' results first
